src/clj/harja/palvelin/integraatiot/api/analytiikka.clj

```diff
@@ -1117,7 +1117,7 @@
     (julkaise-reitti
       http :analytiikka-hae-paikkaukset
       (GET "/api/analytiikka/paikkaukset/:alkuaika/:loppuaika" parametrit
-        (kasittele-get-kutsu db integraatioloki :analytiikka-hae-paikkaukset parametrit
+        (kasittele-get-kutsu db integraatioloki :analytiikka-hae-paikkaukset "analytiikka" parametrit
```

Comment:
Oon just näitä samoja korjannut toisessa analytikkaan liittyvässä pr:ssä :) 
https://github.com/finnishtransportagency/harja/pull/3504

Great minds think alike?